tests/test_fields.py

```diff
@@ -479,7 +489,17 @@ class TestIntegerRangeField(FieldValues):
     Values for `ListField` with CharField as child.
     """
     serializer_class = IntegerRangeSerializer
-
+    valid_instances = [
+        ({'lower': '1', 'upper': 2, 'bounds': '[)'},
+         {'lower': 1, 'upper': 2, 'bounds': '[)'}),
+        ({'lower': 1, 'upper': 2},
+         {'lower': 1, 'upper': 2, 'bounds': None}),
+        ({'lower': 1},
+         {'lower': 1, 'upper': None, 'bounds': None}),
+        ({'upper': 1},
+         {'lower': None, 'upper': 1, 'bounds': None}),
+        ({}, {}),
+    ]
```

Comment:
Should we add `NumericRange` instances here?

Reply:
No need, they are already being tested via test_outputs. I will move these to there as well.